### PR TITLE
PHPLIB-558: Don't inherit read preference in Database::command

### PR DESCRIPTION
**Pull request by alcaeus**
https://jira.mongodb.org/browse/PHPLIB-558

~Same as with https://github.com/mongodb/mongo-php-driver/pull/1144, I've targeted the upcoming minor release for this fix, but can back port to the current stable release for a July patch release.~

### CODE REVIEW

File: tests/Database/DatabaseFunctionalTest.php [hunk 1]
```diff
@@ -100,6 +101,22 @@ public function testCommand()
         $this->assertTrue($commandResult->ismaster);
     }
 
+    public function testCommandDoesNotInheritReadPreference()
+    {
+        if ($this->getPrimaryServer()->getType() !== Server::TYPE_RS_PRIMARY) {
```
Comment (jmikola) [inline tests/Database/DatabaseFunctionalTest.php:106]: What do you think about creating an `isReplicaSet` method in the base `FunctionalTestCase` class (alongside `isShardedCluster`). Then you can also refactor `CreateIndexesFunctionalTest::testCommitQuorumOption` to use the same method.

File: tests/Database/DatabaseFunctionalTest.php [hunk 2]
```diff
@@ -100,6 +101,22 @@ public function testCommand()
...
+            $this->markTestSkipped('Test only applies to replica sets');
+        }
+
+        $this->database = new Database($this->manager, $this->getDatabaseName(), ['readPreference' => new ReadPreference(ReadPreference::RP_SECONDARY)]);
```
Comment (jmikola) [inline tests/Database/DatabaseFunctionalTest.php:110]: I like that this test should also pass even if the replica set has a single primary member :+1:

File: tests/Database/DatabaseFunctionalTest.php (hunk 1 above)
Comment (alcaeus) [reply tests/Database/DatabaseFunctionalTest.php:106]: Done. The second test is only in 1.7, so I'll replace that occurrence when merging up.

**Comment by alcaeus** [comment]
@jmikola I'll ignore the phpcs warning in this PR as tests are stable. I'll fix coding style in a separate pull request.

**Pull request merged**